lib/Backend/FunctionCodeGenJitTimeData.h

```diff
@@ -46,6 +47,9 @@ namespace Js
         // accurate count.
         Field(uint) ldFldInlineeCount;
 
+        // Number of functions passed as arguments to be inlined.
+        Field(uint) callbackInlineeCount;
```

Comment:
We currently have four unused bytes after globalObjTypeSpecFldInfoCount; could this go up there?

Reply:
This member isn't actually needed. I'm going to remove it.